implementations/rust/ockam/ockam_command/src/relay/create.rs

```diff
@@ -53,11 +54,14 @@ pub struct CreateCommand {
     #[arg(long)]
     relay_address: Option<String>,
 
-    /// Whether the relay will be used to relay messages at a project.
-    /// By default, this information will be inferred from the `--at` argument.
+    /// Deprecated
```

Comment:
We should add a warn message to display this. Something like [this](https://github.com/build-trust/ockam/blob/167a0e310cea891669fd3990068ec5f19b3c641a/implementations/rust/ockam/ockam_command/src/node/stop.rs#L42) would work (perhaps we could extract the print logic into a function, similar to [this](https://github.com/build-trust/ockam/blob/d0e14a630956b72cb27dd047c9ae3d9e9dd55a51/implementations/rust/ockam/ockam_command/src/util/mod.rs#L212)). I'd add this logic inside the `parse_args` function.